runtime/pom.xml

```diff
@@ -168,6 +169,11 @@
             <artifactId>grpc-stub</artifactId>
             <version>${grpc.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.ehcache</groupId>
+            <artifactId>sizeof</artifactId>
+            <version>${ehcahce.sizeOf.version}</version>
+        </dependency>
```

Comment:
In the past we did try using ehcache, but the problem was it had a large memory footprint for some objects. So in addition to the running time cost, we should also investigate the memory overhead, this is tricky because its specific to the object structure.

Reply:
yes, this is true.
While the sizeOf instance itself just takes bytes, there are other classes which occupy in MBs. And like you said, it depends on which objects are currently in memory